src/utils.js

```diff
@@ -61,11 +62,16 @@ const encodeHTML = (string) => {
 }
 
 const replaceText = (string, options) => {
+  let defaultFlags = 'g'
+
   if (!options.replaceText) {
     return string
   }
+  if (options.flags && typeof options.flags === 'string') {
```

Comment:
should this throw if `flags` is not a string or undefined?

Reply:
no, keep the default behavior if those conditions are not met

Reply:
why would we want to silently ignore bad input?

Reply:
@cookpete what do you think?

Reply:
If someone is passing a `flags` option, they're already at risk of breakage if they're passing a string, so I don't think we need to worry about that.